app/models/decidim/sdgs/application_record.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
this should not be added by this PR, this should already be in master, can you rebase against master please